pkg/trait/openapi.go

```diff
@@ -270,7 +270,7 @@ func (t *openAPITrait) createNewOpenAPIConfigMap(e *Environment, resource v1.Dat
 		return err
 	}
 	mc.GlobalSettings = data
-	//nolint: staticcheck,nolintlint
+	// nolint: staticcheck,nolintlint
```

Comment:
Btw, with upcoming golangci-lint version nolintlint should invalidate it if it's in the form of `// nolint:`. The correct one should be `//nolint:`.

Reply:
Reverted. My IDE keeps updating it!